etc/icinga2/features-available/influxdb2.conf

```diff
@@ -0,0 +1,27 @@
+/**
+ * The Influxdb2Writer type writes check result metrics and
+ * performance data to an InfluxDB v2 HTTP API
+ */
+
+object Influxdb2Writer "influxdb2" {
+  //host = "127.0.0.1"
+  //port = 8086
+  //organization = "monitoring"
+  //bucket = "icinga2"
+  //auth_token = "0123456789abcdef"
```

Comment:
The `auth_token` isn't hex but base64 (url-safe encoding). I think this snippet should reflect that.